<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>用户管理</string>
    </property>
    <widget class="QTableView" name="userTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>651</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addUserButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>60</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeUserButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>140</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>消息管理</string>
    </property>
    <widget class="QTableView" name="messageTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>781</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>文件管理</string>
    </property>
    <widget class="QTableView" name="fileTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>781</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
